Sources/Core/BoxSDKConfiguration.swift

```diff
@@ -84,14 +90,16 @@ public struct BoxSDKConfiguration {
         retryBaseInterval: TimeInterval? = defaultRetryBaseInterval,
         consoleLogDestination: ConsoleLogDestination? = ConsoleLogDestination(),
         fileLogDestination: FileLogDestination? = nil,
-        clientAnalyticsInfo: ClientAnalyticsInfo? = nil
+        clientAnalyticsInfo: ClientAnalyticsInfo? = nil,
+        callbackURL: String? = nil
     ) throws {
         self.clientId = clientId
         self.clientSecret = clientSecret
 
         self.apiBaseURL = apiBaseURL ?? defaultAPIBaseURL
         self.uploadApiBaseURL = uploadApiBaseURL ?? defaultUploadAPIBaseURL
         self.oauth2AuthorizeURL = oauth2AuthorizeURL ?? defaultOAuth2AuthorizeURL
+        self.callbackURL = callbackURL
```

Comment:
What do you think about setting the callbackURL here where if nothing is passed, the defaultCallbackURL is used like it is for the urls above.

Reply:
I like that idea, it'll make BoxSDK.swift a lot cleaner because then we can just reference `configuration.callbackURL` everywhere instead of having conditionals. I'll try out making that change 👍